include/oneapi/tbb/flow_graph.h

```diff
@@ -250,6 +274,23 @@ class receiver {
         return true;
     }
 
+#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
+    //! Put an item to the receiver and wait for completion
+    bool try_put_and_wait( const T& t ) {
+        d1::small_object_allocator alloc{};
+        d1::wait_context_vertex* msg_wait_context = alloc.new_object<d1::wait_context_vertex>();
```

Comment:
Does it make sense to create it on the stack since it is a blocking call ?

Reply:
Done